client/components/tinymce/plugins/mentions/style.scss

```diff
@@ -0,0 +1,126 @@
+$base-background-color: $white;
+$list-background:       $white;
+$border-color:          lighten( $gray, 20% );
+$light-border-color:    lighten( $gray, 30% );
+$text-color:            $gray-dark;
+$name-color:            $gray;
+$selected-color:        $blue-medium;
+$img-size:              24px;
+$img-border-radius:     50%;
+$border-radius:         3px;
+$box-shadow:            0px 5px 6px rgba( lighten( $gray, 20% ), 0.5 );
+
+.suggestions {
+    position: relative;
+    left: 50%;
+    transform: translate(-50%, 0);
+    margin-top: 5px;
+    color: $text-color;
+    background: $base-background-color;
+    border: 1px solid $border-color;
+    border-radius: $border-radius;
+    box-shadow: $box-shadow;
+    z-index: 20010;
+    font-family: $sans;
+    width: 90%;
+
+    ul {
```

Comment:
We prefer not to use element selectors and instead target elements with classes. You can read more about our CSS guidelines here: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md